src/utils.ts

```diff
@@ -78,41 +100,52 @@ export const CANONICAL_URL_REGEX =
  * which is passed the given URL and returns a promise resolving to a YT canonical URL,
  * from which to derive the Holodex URL from.
  */
-export async function getHolodexUrl(
-  url: string | undefined,
-  findCanonicalUrl: (url: string) => Promise<string | null>
-) {
-  if (url) {
-    if (HOLODEX_URL_REGEX.test(url)) {
-      return null;
-    }
-    const videoMatch = url.match(VIDEO_URL_REGEX);
+export async function getHolodexUrl(url: string | undefined, isMultiview: boolean) {
```

Comment:
The if/else flow took me a second to follow through, if we can add a few comments to make it easier to follow it would be nice! (Not required change)